test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -165,17 +271,21 @@ define([
         // row ID: may be specified in job.meta.row_id, e.g. if the job is a retry
         expect(row.id).toEqual(`job_${job.meta && job.meta.row_id ? job.meta.row_id : job.job_id}`);
 
-        // status details cell
-        const logViewEl = row.querySelector(`.${cssBaseClass}__cell--log-view`);
-        expect(logViewEl.textContent).toContain('Status details');
+        // type
```

Comment:
no more status details cell; now have the import type and the output object name